tests/test-validate-app-receipt.js

```diff
@@ -127,9 +135,9 @@ describe('fxpay.validateAppReceipt()', function() {
 
     var validator = new helper.ReceiptValidator();
 
-    fxpay.validateAppReceipt(function(error) {
-      done(error);
-    });
+    fxpay.validateAppReceipt().then(function() {
+      done();
```

Comment:
That doesn't work because the first argument to a successful then is ``productInfo``. It's not shown here because jshint will blow up.